### ISSUE TITLE
Please add host key display somewhere in connection details

### ISSUE THREAD

**Issue by ssten** [FreeRDP/Remmina#1403]
> Client (OS name and version): Gentoo GNU/Linux amd64 profile
> Remmina version (remmina --version): net-misc/remmina (1.2.0_rc24@07.11.2017)
> Remmina - Version 1.2.0-rcgit-24 (git rcgit-24)
> Desktop environment: XFce 4.12

Remmina identifies server by it's key at least for ssh and rdp connections.
For **ssh** connections it use standard user config file *~/.ssh/known_hosts*, so I can easely see host key.
For **rdp** it stores keys somehere in it's profile.

It will be useful, if remmina will display stored host key (or warning, if missed) in connection details.

**Comment by antenore** [comment]
It should be already the case if you execute remmina from the command line, isn't it?
The known_host file is managed by FreeRDP and it's usually stored under ```~/.config/freerdp/known_hosts2```

Can you confirm you see the warning/errors in the terminal?

**Comment by ssten** [comment]
Maybe it depends on some build options, but just for now I see no warning/error messages in the terminal.
Even when connecting (via **ssh**) to server with new (i.e. unknown) host key.

**Comment by antenore** [comment]
Bizarre

Opening the Debug window you should see everything at least. It's possible that you have to change the logging level in the general SSH preferences (in the Remmina main menu -> preferences, under the SSH tab). Anything useful there?

I'll improve the SSH notifications in the next releases.

That said, consider that the Remmina SSH protocol plugin, it's still really rudimentary and it's not meant to replace the OpenSSH one.
In Remmina there are not full time developers unlucky. I'll do my best, but I cannot guarantee anything.

**Comment by ssten** [comment]
Is loglevel settings a runtime parametr or I need to restart remmina to take effect?
On connection I see only two warnings:

> Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

> (remmina:12135): Gtk-WARNING \*\*: Allocating size to GtkScrollbar 0x174ca50 without calling gtk_widget_get_preferred_width/height(). How does the code know the size to allocate?

Thank you for your efforts!
I just want to help you to improve remmina and make it perfect.

**Comment by antenore** [comment]
No, you must lunch the debug window, it's in the main Remmina menu and then just re-lunch your ssh connection

**Comment by ssten** [comment]
Excuse me long response.
First time I've little bit confused with searching debug output.

But in debug session I see no info about server's key. Just:

> [SSH] ssh_knownhosts_algorithms: server server.domain.tld:22 has ssh-rsa in known_hosts

This may be a kind of right behaviour.
But it will be good improvement to display stored host key (or it's fingerprints) somewhere in connection details.

P.S. One more strange message in debug session:
> [SSH] ssh_analyze_banner: We are talking to an OpenSSH client version: 4.5 (40500)

Maybe not **client** but **server**?

**Comment by antenore** [comment]
> [SSH] ssh_analyze_banner: We are talking to an OpenSSH client version: 4.5 (40500)

This comes from libssh and it should be reported to them.

I'm thinking where to show those information, I'm afraid to make the interface too bloated.

One thing I was thinking of it's something similar to virt-manager, where we can see the status of the connection, how many resources it's using and all the parameters of the connection, like what the user has deliberately have chosen and what has been set by Remmina.

It'll take some time to do this.

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @ssten
